sdk/python/feast/feature_store.py

```diff
@@ -168,6 +170,11 @@ def __init__(
         registry_config = self.config.get_registry_config()
         if registry_config.registry_type == "sql":
             self._registry = SqlRegistry(registry_config, None)
+
+            if is_feast_apply:
```

Comment:
@IsraelAgundis 
We want to build the proto registry even if we're not in apply.